protocols/cosi/packets.go

```diff
@@ -0,0 +1,101 @@
+package cosi
+
+import (
+	"github.com/dedis/cothority/lib/cosi"
+	"github.com/dedis/cothority/lib/sda"
+	"github.com/satori/go.uuid"
+)
+
+// This files defines the structure we use for registering to the channels by
+// SDA.
+
+// The main messages used by CoSi
+
+// Broadcasted message initiated and signed by proposer
+type CosiAnnouncement struct {
+	// From = TreeNodeId in the Tree
+	From uuid.UUID
```

Comment:
Yeah I agree, I did that within the very firsts versions of SDA but we dropped it later when switched to UUID. You'd like stg : TreeNodeID, EntityID, ProtocolID etc... ?